[Validator] Document the DivisibleBy constraint

colinodell opened the pull request:
Documentation for the new ~`MultipleOf`~ `DivisibleBy` constraint proposed in https://github.com/symfony/symfony/pull/28069

colinodell committed:
Document the DivisibleBy constraint

colinodell commented:
The constraint has been renamed, so I've updated the documentation accordingly.

javiereguiluz reviewed (approved):
@colinodell thanks a lot for providing this new constraint and for documenting it too. Sadly there are some merge conflicts. Could you please fix them? Thanks!

colinodell commented:
Thanks for the heads-up @javiereguiluz! Looks like the `Range` constraint was being added in the same spot in the docs :upside_down_face:  I have squashed my two commits together and rebased them on top of `master`.
